src/plugins/catalog/api.go

```diff
@@ -89,7 +100,12 @@ func (catalogApi *CatalogApi) CreateCatalog(ctx *gin.Context) {
 		return
 	}
 
-	catalogApi.Save(&catalog)
+	catalog.UserId = account.(auth.Account).ID
```

Comment:
UserId  => AccountId  would be great as we do not have such model named "User", we use "Account"